pkg/hub/taints/doc.go

```diff
@@ -0,0 +1 @@
+package taints
```

Comment:
why need this？

Reply:
Looks like it should be deleted, because no description was added.

Reply:
we usually add comments here to explain the function of this package